Tests/Admin/AdminTest.php

```diff
@@ -1080,16 +1138,22 @@ public function testGetTemplate2()
 
     public function testGetIdParameter()
     {
-        $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
+        $postAdmin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
+
+        $this->assertSame('id', $postAdmin->getIdParameter());
+        $this->assertFalse($postAdmin->isChild());
 
-        $this->assertSame('id', $admin->getIdParameter());
-        $this->assertFalse($admin->isChild());
+        $commentAdmin = new CommentAdmin('sonata.post.admin.comment', 'Application\Sonata\NewsBundle\Entity\Comment', 'SonataNewsBundle:CommentAdmin');
```

Comment:
Please linebreak this.

Reply:
https://dev.to/greg0ire/why-i-keep-pestering-everyone-about-long-lines